public/app/themes/clarity/inc/content-filter/search.php

```diff
@@ -75,55 +75,81 @@ function load_events_filter_results()
     die();
 }
 
+function get_args() {
+    $oAgency = new Agency();
+    $activeAgency = $oAgency->getCurrentAgency();
+    $post_per_page = 10;
+
+    $valueSelected = isset($_POST['valueSelected']) ? sanitize_text_field($_POST['valueSelected']) : ''; 
+    $before = '';
+    $after = '';
+
+    if ($valueSelected) {
+        preg_match('/&after=([^&]*)&before=([^&]*)/', $valueSelected, $matches);
+        $after = date('Y-m-d', strtotime($matches[1]));
+        $before = date('Y-m-d', strtotime($matches[2]));
+    }
+
+    $nextPageToRetrieve = isset($_POST['nextPageToRetrieve']) ?  sanitize_text_field($_POST['nextPageToRetrieve']) : '';
+    $offset = $nextPageToRetrieve ? ($nextPageToRetrieve - 1) * $post_per_page : 0;
```

Comment:
Hi @EmilyHazlehurst, first, well done on the work here, I just wanted to check; has this been tested locally?

There has been quite some update so I wanted to check for errors before reviewing and this line flagged up in Sentry.

[TypeError Unsupported operand types: string - int](https://ministryofjustice.sentry.io/issues/5198690924/?environment=development-damien&project=4506870789505024&query=is%3Aunresolved&referrer=issue-stream&statsPeriod=14d&stream_index=0)

I would suggest wrapping the calculation in parenthesis to fix it but as always, will leave the decision up to you 👍 

```php
$offset = $nextPageToRetrieve ? (($nextPageToRetrieve - 1) * $post_per_page) : 0;
```

Reply:
Hi @wilson1000, I've updated this to use your suggestion. 

I have tested this locally; oddly I don't seem to get the error you described. 
Where do you generally look for error messages? I would have expected them to be printed to the page or the php container logs, but perhaps I'm missing a testing step that's Wordpress specific.

Reply:
Hiya.
It's in the link, it fired in Sentry:
https://ministryofjustice.sentry.io/issues/5198690924/?environment=development-damien&project=4506870789505024&query=is%3Aunresolved&referrer=issue-stream&statsPeriod=14d&stream_index=4

Reply:
... that means the software running in the container is firing the error. When developing,  try to always run the containers as this will give you a preproduction environment to dev in - if you have your sentry ID set up in `.env` you will get an application error in Sentry 👍